handbook/customer-success/README.md

```diff
@@ -34,6 +34,22 @@ Before a routine customer call, the CSM prepares an agenda including the followi
 5. Provide updates to open feature requests (can be done monthly or quarterly)
 6. Provide updates to open bug reports
 
+### Generate an expansion opportunity in Salesforce
+Customer Success Manager's (CSM's) are responsible for developing customer expansion opportunities that are not being worked on in conjunction with an Account Executive (AE). An AE will be assigned by the Chief Revenue Officer (CRO) for large scale expansion opportunities such as bringing on a new Fleet use case or bringing on a new group of hosts to an existing Fleet use case. CSM's manage expansion opportunities for things like host count increase for customer growth and price increases on renewals. Discuss examples of these scenarios with your manager to learn more.
+
+Take the steps below when creating an expansion opportunity in Salesforce:
+
+1. Navigate to the customer account record in Salesforce.
+2. Scroll down to the section labeled "Upcoming renewal (± expansions)".
```

Comment:
```suggestion
2. Scroll down to the "Upcoming renewal (± expansions)" table (in the "Customer" section) and click "New".
```
- The table is named "Upcoming renewal (± expansions)" but the section is "Customer"
- Potential to cut some words and combine steps 2 and 3.